src/os/bluestore/BlueStore.cc

```diff
@@ -8392,6 +8429,65 @@ int BlueStore::queue_transactions(
     dout(10) << __func__ << " new " << osr << " " << *osr << dendl;
   }
 
```

Comment:
when do shard here by m_tran_worker_num, requests in same PG may go different tran_workers, then the following process may break the sequence of the request for the PG, how to guarantee the request sequence in the same PG?